integration/compactor_test.go

```diff
@@ -229,3 +229,10 @@ func (s sample) Type() chunkenc.ValueType {
 		return chunkenc.ValFloat
 	}
 }
+
+func must[T any](v T, err error) T {
+	if err != nil {
+		panic(err)
```

Comment:
[non blocking] I see this pattern used quite a lot in this PR. I would have preferred avoiding the panic and using `require.NoError()` instead. The `must()` could take `testing.T` in input.

Reply:
Also, have you considered defining it in `pkg/util/test`?

Reply:
I have considered both things, but they didn't convince me so I went for _little copying is better than a little dependency_.

I'll merge this as is, and then will send a separate PR with a proposal to extract it.